test/sql/window/test_streaming_lead_lag.test

```diff
@@ -2,6 +2,8 @@
 # description: Streaming window functions
 # group: [window]
 
+require vector_size 2048
```

Comment:
Could we test this with vsize=2? There are a number of gnarly corner cases with small vectors in the algorithm - I think a vsize=2 test would be very beneficial to make sure nothing sneaks through the cracks.

Reply:
Perhaps we can set `MAX_BUFFER` to `MinValue<idx_t>(STANDARD_VECTOR_SIZE, 2048)` instead so we can still use the streaming window for lower vector sizes?

Reply:
I'm just setting it to a fixed value. That seems more predictable.